src/schema/home/results.js

```diff
@@ -108,6 +108,33 @@ const moduleResults = {
       sort: "-position",
     })
   },
+  similar_to_saved_works: ({
+    rootValue: { savedArtworksLoader, similarArtworksLoader, artworksLoader },
+  }) => {
+    return savedArtworksLoader({
+      size: RESULTS_SIZE,
+      sort: "-position",
+    }).then(works => {
+      return similarArtworksLoader(
+        { artwork_id: map(works, "_id").slice(0, 7) }
+      ).then(ids => {
+        return artworksLoader({ ids })
+      })
+    })
+  },
+  similar_to_recently_viewed: ({
+    rootValue: { meLoader, similarArtworksLoader, artworksLoader },
+  }) => {
+    return meLoader().then(({ recently_viewed_artwork_ids }) => {
+      if (recently_viewed_artwork_ids.length === 0) {
+        return []
+      }
+      const ids = recently_viewed_artwork_ids.slice(0, 7)
+      return similarArtworksLoader({ artwork_id: ids }).then(similarIds => {
+        return artworksLoader({ similarIds })
```

Comment:
Same as above, I don't think we need `artworksLoader` at all here? Since you already get the artworks back...

Also, `({ similarIds })` will wind up passing in that as a named param, so: `/api/v1/artworks?similarIds=...` so probably not what you want. But yea, I think we can just drop the `artworksLoader` stuff.

Reply:
So yea, now that I realize we need `artworksLoader`, this is a bug, the param needs to be called `ids`. Otherwise `/api/v1/artworks` will ignore a param called `similarIds` and just return random artworks.

Reply:
Oh I see! I was wondering why I was seeing weird data here.

Reply:
`ids` was giving a warning as it's a name clash with the constant in the parent scope - that's why I changed it. will change the name of the constant instead.